build.sbt

```diff
@@ -335,6 +336,7 @@ lazy val codegenScalaCompilationTest = project
     akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala),
     (publish / skip) := true,
     name := "akkaserverless-codegen-scala-compilation-tests",
+    Compile / PB.protoSources ++= (codegenProtoAnnotations / Compile / PB.protoSources).value,
```

Comment:
TIL that to get the proto sources in a scala project one need to be explicit about it. Seems not to need for the Java counterpart.